tokenizers/src/utils/proc_macros/src/lib.rs

```diff
@@ -0,0 +1,180 @@
+extern crate proc_macro;
+use ::proc_macro::*;
+
+// Original: https://crates.io/crates/macro_rules_attribute
```

Comment:
since https://crates.io/crates/macro_rules_attribute was not v1 (currently, v0.0.2) & the source code was very small, decided to just